docs/topics/proc_creating-nodejs-app.adoc

```diff
@@ -102,6 +102,34 @@ $ npm install nodeshift --save
 +
 The `openshift` script uses `nodeshift` to deploy the application to OpenShift.
 
+. _Optional_: Add a `files` section in `package.json`.
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+{
+  "name": "myApp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    ...
+  },
+  "files": {
+    "package.json",
+    "app.js"
+  }
+  ...
+}
+----
++
+The `files` section tells `nodeshift` what files to include when deploying to OpenShift. If the `files` section is not specified, `nodeshift` will send the entire current directory, excluding:
```

Comment:
Multiple sentences on single line :-)

Also, more importantly -- can the entries in the `files` section contain wildcards? I'd expect to find out in this description. @lholmquist?

Reply:
like how wild are you thinking.

if you want to do a directory, for example, `lib`, you can just specificy `lib` and that will get everything in that directory including sub directories.

this is the module we use to do that archiving: https://github.com/npm/node-tar
and we pass in an array of files to the create method.

if there is some crazy pattern matching stuff that someone needs, then i'd imaging they've written their app wrong

Reply:
I think that's perfectly enough, but from the present text, I didn't get the impression that I can include directory names as well. I actually got the opposite impression -- that I need to list all the files.

Reply:
that is true.  lets tell @rhoads-zach to add it  :)